packages/utilities/fast-web-utilities/src/dom.spec.ts

```diff
@@ -9,206 +11,215 @@ import {
                     keyCode: 38,
                 } as KeyboardEventInit)
             )
-        ).toBe(KeyCodes.arrowUp);
+        ).to.equal(KeyCodes.arrowUp);
         expect(
             getKeyCode(
                 new KeyboardEvent("keydown", {
                     keyCode: 40,
                 } as KeyboardEventInit)
             )
-        ).toBe(KeyCodes.arrowDown);
+        ).to.equal(KeyCodes.arrowDown);
         expect(
             getKeyCode(
                 new KeyboardEvent("keydown", {
                     keyCode: 13,
                 } as KeyboardEventInit)
             )
-        ).toBe(KeyCodes.enter);
+        ).to.equal(KeyCodes.enter);
         expect(
             getKeyCode(
                 new KeyboardEvent("keydown", {
                     keyCode: 32,
                 } as KeyboardEventInit)
             )
-        ).toBe(KeyCodes.space);
+        ).to.equal(KeyCodes.space);
         expect(
             getKeyCode(
                 new KeyboardEvent("keydown", {
                     keyCode: 9,
                 } as KeyboardEventInit)
             )
-        ).toBe(KeyCodes.tab);
+        ).to.equal(KeyCodes.tab);
     });
 
-    test("should correctly handle keyboard events with `which` values", () => {
+    it("should correctly handle keyboard events with `which` values", () => {
+        // Since `which` is deprecated, it can no longer be set using KeyboardEvent
         expect(
-            getKeyCode(
-                new KeyboardEvent("keypress", {
-                    which: 39,
-                } as KeyboardEventInit)
-            )
-        ).toBe(KeyCodes.arrowRight);
+            getKeyCode({
+                which: 39,
+                keyCode: 39,
+                charCode: 39,
+            } as any)
+        ).to.equal(KeyCodes.arrowRight);
         expect(
-            getKeyCode(
-                new KeyboardEvent("keypress", {
-                    which: 37,
-                } as KeyboardEventInit)
-            )
-        ).toBe(KeyCodes.arrowLeft);
+            getKeyCode({
+                which: 37,
+                keyCode: 37,
+                charCode: 37,
+            } as any)
+        ).to.equal(KeyCodes.arrowLeft);
         expect(
-            getKeyCode(
-                new KeyboardEvent("keypress", {
-                    which: 38,
-                } as KeyboardEventInit)
-            )
-        ).toBe(KeyCodes.arrowUp);
+            getKeyCode({
+                which: 38,
+                keyCode: 38,
+                charCode: 38,
+            } as any)
+        ).to.equal(KeyCodes.arrowUp);
         expect(
-            getKeyCode(
-                new KeyboardEvent("keypress", {
-                    which: 40,
-                } as KeyboardEventInit)
-            )
-        ).toBe(KeyCodes.arrowDown);
+            getKeyCode({
+                which: 40,
+                keyCode: 40,
+                charCode: 40,
+            } as any)
```

Comment:
Out of curiosity..why the change to `any` and the move away from `KeyboardEvent`?

Reply:
Because `KeyboardEvent` was giving me type errors since `which` is deprecated.